STEP_Project/popup.js

```diff
@@ -12,6 +12,8 @@ const PAGE_TYPES = {
   checkTabUrl();
   addRemoveButton = document.getElementById("addRemove");
   injectContent(addRemoveButton);
+  let selectButton = document.getElementById("select");
+  injectSelectContent(selectButton);
```

Comment:
We are not actually injecting the select button here, right ? We are just adding event listener to an already existing button. Maybe we should rename this to something appropriate.

Reply:
Renamed it to selectListener